tests/bookmarks_and_history/test_bookmark_website_via_star_button.py

```diff
@@ -13,6 +13,7 @@ def test_bookmark_website_via_star(driver: Firefox):
     # instantiate object
     nav = Navigation(driver)
     panel = PanelUi(driver)
+    # random comment
```

Comment:
meant to be here?

Reply:
LOOOOL, the change that started it all. No, that should be removed.